grandnode/grandnode2
# Login with email code

Resolves #172 
Type: feature

## Issue
Adds the ability to login to the system using an e-mail code instead of the password

## Solution
Adds a link on the Login page that allows the user to send themself an e-mail that contains a link to log the user on directly to the system.

## Breaking changes
I have edited both InstallDataMessageTemplates.cs & DefaultLanguage.xml, so upgrade scripts will need to be provided to update this. I checked the migrations folder and they seemed to be specific to version upgrades. Was unsure where the code for this upgrade should go.

## Testing
1. Visit frontpage of shop without being logged into an account.
2. Head to the Login page
3. Click "Login with E-mail Code"
4. Enter your e-mail address
5. Head to your e-mail and click the link provided
6. You should now be logged into the system.
